site/enterprise-faq.html

```diff
@@ -0,0 +1,55 @@
+---
+layout: default
+title: Enterprise FAQ
+description: Frequently asked questions about our enterprise sales
+---
+<section class="purple hero">
+  <div class="container">
+    <div class="row">
+      <div class="col-xs">
+        <h1>We don't have a sales team</h1>
+        <p>
+          Go ahead and sign up for one of our <a href='/pricing'>editions</a> and let us know if there's something specific you need.
+        </p>
+        <p>
+           <a href='/pricing' class="button brochure orange text-larger">Sign up for an edition</a>
+        </p>
+      </div>
+    </div>
+</section>
+
+{% include _customers.liquid %}
+
+<section class="white hero">
+  <div class="container">
+    <div class="row">
+      <div class="col-xs">
+        <h1>More questions?</h1>
+        <h3>Why don't you have a sales team?</h3>
+        <p>
+          We got tired of dealing with procurement managers who want to endlessly negotiate pricing and legal terms for no reason until one or both parties ultimately die of exhaustion or boredom. We prefer to spend our time making great software, so we fired our sales team and instead have spent the budget on growing the product team. It may sounds counterintuitive to allocate resources to improve the product we're selling, rather than to spend time making PDFs selling it, but we've had pretty reasonable success with this strategy.
```

Comment:
Another point we could make is that "we are a bunch of engineers, and we like the idea of great software selling itself instead of a a sales team trying to convince you."